workflows/pipe-common/scripts/autoscale_sge.py

```diff
@@ -617,6 +614,9 @@ def scale_up(self, resource):
 
     def _launch_additional_worker(self, instance):
         Logger.info('Launch additional worker.')
+        parent_run = self.api.load_run(self.parent_run_id)
+        worker_launch_system_params = self.fetch_worker_launch_system_params(parent_run)
```

Comment:
It seems like a good idea to move this initialization logic to the beginning of *main* method. It makes the whole autoscaling launch consistent independently of changes in the system preference.